mozilla-release/browser/confvars.sh

```diff
@@ -35,8 +35,8 @@ MOZ_SERVICES_COMMON=1
 MOZ_SERVICES_CRYPTO=1
 MOZ_SERVICES_HEALTHREPORT=1
 MOZ_SERVICES_METRICS=1
-MOZ_SERVICES_SYNC=1
```

Comment:
Better to redirect this to our URL and put a dummy handler for it. Never knows when it could be useful for us